src/hosted.js

```diff
@@ -95,7 +95,11 @@
     function load(Bramble) {
         Bramble.load("#bramble",{
             url: "index.html",
-            useLocationSearch: true
+            useLocationSearch: true,
+            enableCollaboration: true,
```

Comment:
Removed it [here](https://github.com/mozilla/brackets/pull/828/commits/17811c94bb09bdaf42f13e5126b384326eee251f)